isotovideo

```diff
@@ -165,12 +165,18 @@ sub kill_commands {
         my $url       = "http://127.0.0.1:$cmd_srv_port/$job_token/broadcast";
         diag('isotovideo: informing websocket clients before stopping command server: ' . $url);
 
-        my $ua = Mojo::UserAgent->new(request_timeout => 15);
-        my $tx = $ua->post($url, json => {stopping_test_execution => $reason});
+        my $timeout = 15;
+        my $ua      = Mojo::UserAgent->new(request_timeout => $timeout);
+        my $tx      = $ua->post($url, json => {stopping_test_execution => $reason});
         try {
             my $res = $tx->result;
             if ($res->code != 200) {
-                diag('isotovideo: unable to inform websocket clients about stopping command server: ' . $res->to_string);
+                my $error_message = $res->to_string;
+                diag('isotovideo: unable to inform websocket clients about stopping command server: ' . $error_message);
+                if ($error_message =~ qr/.*timeout.*/i) {
+                    diag("This is likely NOT related to the job possibly exceeding its timeout as it takes max. $timeout seconds. This indicates that the command server is under heavy load or has already terminated.");
+                }
+                diag('The command server might have already been killed by the worker after the user has aborted the job or the job timeout has been exceeded.');
```

Comment:
This would be a bit confusing, don't you think?... I mean, if the message above is shown, and also this one... it would look a bit awkward